deb/build

```diff
@@ -32,10 +40,27 @@ pkgmaint=$(echo "`git config user.name` <`git config user.email`>")
 
 # package name
 pkgname=libdrizzle-redux
-so_filename=libdrizzle-redux.so
 
-version_number=`find ./install/usr/lib/ ! -type l -name "$so_filename.*"`
+# library .so name
+so_libname=$pkgname.so
+
+# get the api level for the package from the cmd args
+api_level=""
+if [ $# -eq 1 -a ! -z "$1" ]; then
+    api_level=$1
+fi
+
+# the package name used in the distribution
+dist_pkgname=$pkgname$api_level
 
+# the so library name used in the distribution
+dist_so_libname=$dist_pkgname.so
+
+# relative path to the directory with library files to install
+libdir="./install/usr/lib"
+
+# get the ABI library version from the generated .so file
+version_number=`find $libdir ! -type l -name "$so_libname.*"`
 if [ $(echo $version_number | wc -l) != 1 ]; then
```

Comment:
Not part of this PR, but this is a very weird way to check if a file is found or not :thinking: